packages/patternfly-4/react-core/src/components/Alert/Alert.js

```diff
@@ -60,19 +64,31 @@ const Alert = ({
     </React.Fragment>
   );
 
-  const customClassName = css(styles.alert, getModifier(styles, variant, styles.modifiers.info), className);
-
+  const customClassName = css(
+    styles.alert,
+    getModifier(styles, variant, styles.modifiers.info),
+    className,
+    (toast && 'pf-m-live')
```

Comment:
is this modifier available through styles.modifiers.live?

Reply:
Not at the moment

<img width="742" alt="Screen Shot 2019-04-15 at 10 01 04 AM" src="https://user-images.githubusercontent.com/5942899/56155926-0ffee780-5f8a-11e9-80cd-f470d01e96f3.png">